backend/infrahub/core/diff/conflict_transferer.py

```diff
@@ -0,0 +1,75 @@
+from .combiner import DiffCombiner
+from .model.path import (
+    EnrichedDiffAttribute,
+    EnrichedDiffNode,
+    EnrichedDiffRelationship,
+    EnrichedDiffRoot,
+    EnrichedDiffSingleRelationship,
+)
+
+
+class DiffConflictTransferer:
```

Comment:
new component to move conflicts from an earlier diff to a later one when completely recalculating